tasks/create_target_vm/03_hosted_engine_final_tasks.yml

```diff
@@ -384,6 +384,10 @@
     yum:
```

Comment:
Is this the only place we use yum? If elsewhere, can still fail anywhere (e.g. due to cron checks etc.).

Reply:
[appliance installation](https://github.com/oVirt/ovirt-ansible-hosted-engine-setup/blob/master/tasks/install_appliance.yml#L7 ) and [ovirt-hosted-engine-setup installation](https://github.com/oVirt/ovirt-ansible-hosted-engine-setup/blob/master/tasks/install_packages.yml#L7) are already protected with the same logic.

In ansible 2.8 we can also rely on lock_timeout parameter but currently we are requiring only 2.7.